CodePush.js

```diff
@@ -135,6 +135,7 @@ async function syncNative(syncOptions = null, syncStatusChangeCallback, download
   }
 
   try {
+    await CodePush.notifyApplicationReady();
```

Comment:
We need to remove all usages of `notifyApplicationReady` method from the native API.

Reply:
@sergey-akhalkov `CodePush.notifyApplicationReady();` will call method from CodePushNativeModule.

Reply:
Got it, thanks.